File: discord-music-bot/src/main/java/net/masterzach32/swagbot/api/R8Ball.java
package net.masterzach32.swagbot.api;

import com.mashape.unirest.http.Unirest;
import com.mashape.unirest.http.exceptions.UnirestException;

public class R8Ball {
	
	private String response;

	public R8Ball() {
		try {
			response = Unirest.get("https://apis.rtainc.co/twitchbot/8ball").asString().getBody();
		} catch (UnirestException e) {
			e.printStackTrace();
		}
	}
	
	public String getResponse() {
		return response;
	}
}

File: discord-music-bot/src/main/java/net/masterzach32/swagbot/api/R8Ball.kt
package net.masterzach32.swagbot.api

import com.mashape.unirest.http.Unirest
import com.mashape.unirest.http.exceptions.UnirestException

class R8Ball {

    var response: String? = null
        private set

    init {
        try {
            response = Unirest.get("https://apis.rtainc.co/twitchbot/8ball").asString().body
        } catch (e: UnirestException) {
            e.printStackTrace()
        }

    }
}